src/IntoleranceSimulator.R

```diff
@@ -17,49 +17,60 @@ require(MASS)
 
 # hard-coded for quick prototyping
 
-protfile = 'structures/AF-P68133-F1-model_v2.pdb'
+protfile = '../structures/AF-P68133-F1-model_v2.pdb'
 
 pdb = read.pdb(protfile)
 
 # calculate residues pairwise distance matrix
-distmat = dm(pdb)
+#distmat = dm(pdb)

```

Comment:
Is this just temporarily commented out, or is this for the final version?

Reply:
It is final, I left it to remember how to NOT calculate the distance matrix. Is that bad practice ?

Reply:
I would remove it, since it's confusing as you read through it. Generally comments would be reserved to explain the intention of a command